Yosemite/YosemiteTests/Stores/MediaStoreTests.swift

```diff
@@ -474,6 +474,63 @@ final class MediaStoreTests: XCTestCase {
         let error = try XCTUnwrap(result.failure as? DotcomError)
         XCTAssertEqual(error, .unauthorized)
     }
+
+    // MARK: test cases for `MediaAction.updateProductIDToWordPressSite`
+
+    /// Verifies that `MediaAction.updateProductIDToWordPressSite` returns the expected response.
```

Comment:
nit:
```suggestion
    /// Verifies that `MediaAction.updateProductID` returns the expected response.
```

Reply:
Missed fixing this before merging. 🙏 
I have covered this in https://github.com/woocommerce/woocommerce-ios/pull/7175